src/presentation/service/services.go

```diff
@@ -0,0 +1,470 @@
+package service
+
+import (
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/alessio/shellescape"
+	"github.com/speedianet/os/src/domain/dto"
+	"github.com/speedianet/os/src/domain/useCase"
+	"github.com/speedianet/os/src/domain/valueObject"
+	voHelper "github.com/speedianet/os/src/domain/valueObject/helper"
+	infraEnvs "github.com/speedianet/os/src/infra/envs"
+	internalDbInfra "github.com/speedianet/os/src/infra/internalDatabase"
+	scheduledTaskInfra "github.com/speedianet/os/src/infra/scheduledTask"
+	servicesInfra "github.com/speedianet/os/src/infra/services"
+	vhostInfra "github.com/speedianet/os/src/infra/vhost"
+	mappingInfra "github.com/speedianet/os/src/infra/vhost/mapping"
+	serviceHelper "github.com/speedianet/os/src/presentation/service/helper"
+)
+
+type ServicesService struct {
+	persistentDbService *internalDbInfra.PersistentDatabaseService
+}
+
+func NewServicesService(
+	persistentDbService *internalDbInfra.PersistentDatabaseService,
+) *ServicesService {
+	return &ServicesService{
+		persistentDbService: persistentDbService,
+	}
+}
+
+func (service *ServicesService) Read() ServiceOutput {
+	servicesQueryRepo := servicesInfra.NewServicesQueryRepo(service.persistentDbService)
```

Comment:
Being used in every method, so this should be where?

Reply:
Done.